newIDE/app/src/MainFrame/index.js

```diff
@@ -578,15 +583,48 @@ const MainFrame = (props: Props) => {
       ].includes(kind)
         ? `${kind} ${name}`
         : kind;
+
+      let customIconUrl = '';
+      if (
+        (kind === 'events functions extension' ||
+          kind === 'events functions extension') &&
```

Comment:
It's the same condition in the OR, I guess you wanted to test for another value?